source/collection-management/tests/pre_and_post_images-collection.yml

```diff
@@ -0,0 +1,87 @@
+description: "pre_and_post_images-collection"
+
+schemaVersion: "1.9"
```

Comment:
Does anything actually require schema 1.9 here? I think this could just use 1.0, unless I'm missing something.

The `createCollection` and `modifyCollection` operations shouldn't factor into this. Drivers must raise errors if operations are unsupported, and those were never properly introduced in a spec/schema version.

Reply:
No I thought maybe the new `modifyCollection` operation warranted a new schema version, but it sounds like that is not the case. Switched back to 1.0.